src/service/performance-impl.js

```diff
@@ -30,6 +30,12 @@ import {whenDocumentComplete} from '../document-ready';
  */
 const QUEUE_LIMIT = 50;
 
+/** @const {string} */
+const VISIBILITY_CHANGE_EVENT = 'visibilitychange';
```

Comment:
Not sure what https://github.com/ampproject/amphtml/blob/master/src/service/document-state.js#L52 is doing but the name of the event seems to be dynamic in certain cases.

Reply:
I think certain browsers have a different event name, but this is a Chrome-only feature so it shouldn't be necessary.